content/docs/esc/vs/infisical.md

```diff
@@ -31,32 +31,32 @@ Infisical is a secrets management tool that provides a centralized platform for
 
 ## Pulumi ESC vs. Infisical: Similarities {#similarities}
 
-Like Infisical, Pulumi ESC is a secrets manager for cloud applications and infrastructure. In both ESC and Infisical, secrets can be stored and accessed through a CLI, SDK, or editor interface. Granular access controls can be implemented across all secrets.
+Like Infisical, Pulumi ESC is a secrets manager for cloud applications and infrastructure. In both ESC and Infisical, secrets can be stored and accessed through a CLI, SDK, or Web editor interface. Granular access controls can be implemented across all secrets.
 
 ## Pulumi ESC vs. Infisical: Key Differences {#differences}
 
-There are a couple of fundamental differences between Infisical and Pulumi ESC. First, ESC and Infisical differ in that Infisical can only add and manage secrets stored in Infiniscal.  ESC supports pulling and centralizing the management of secrets from most secrets managers. Second, Infisical only stores secrets, whereas ESC stores environments, secrets, and configurations. Third, Infiniscal can only store static secrets while ESC can dynamically generate cloud provider credentials for AWS, Azure, and Google Cloud. Fourth, Infiniscal has limited composablity in its environments while ESC has full hierarchical inheritance between its environments.
+There are a couple of fundamental differences between Infisical and Pulumi ESC. First, ESC and Infisical differ in that Infisical can only add and manage secrets stored in Infisical. ESC adopts a truly open ecosystem approach, allowing you to pull secrets stored in most secret and password managers during runtime and use them anywhere. This reduces the time to take full inventory of your secrets across your organization , enabling you to utilize the best solution for the purpose.  Second, the composability and hierarchical nature of ESC offers a familiar and powerful experience for users, reducing duplication and allowing them to get up and running quickly. Third, ESC takes a software engineering approach to versioning with ability to add tags and import specific collections of secrets and configuration via those tags, similar to Docker. Fourth, ESC takes a secure limited privilege path to provisioning dynamic short-term credentials compared to Infisical.
```

Comment:
My theory is that its actually better to mention what Infiniscal's limitations are rather than what ESC can do. ESC's features is highlighted well in all our ESC focused materials. But What Infiniscal (or competitor) can't do or can only do is really not mentioned anywhere on the internet so its more valuable content because its opinionated.

Reply:
Sure. Effectively we can flip the benefits and call them as limitations.